services/core-api/app/api/dams/dto.py

```diff
@@ -1,23 +1,22 @@
 from flask_restx import fields
-
-from app.api.dams.models.dam import DamType, OperatingStatus, ConsequenceClassification
 from app.extensions import api
 
 DAM_MODEL = api.model('Dam', {
     'dam_guid': fields.String(required=True, description='The unique identifier of the dam'),
     'mine_tailings_storage_facility_guid': fields.String,
-    'dam_type': fields.String(enum=DamType, attribute='dam_type.name'),
+    'dam_type': fields.String,
     'dam_name': fields.String,
     'latitude': fields.Float,
     'longitude': fields.Float,
-    'operating_status': fields.String(enum=OperatingStatus, attribute='operating_status.name'),
-    'consequence_classification': fields.String(enum=ConsequenceClassification, attribute='consequence_classification.name'),
+    'operating_status': fields.String,
+    'consequence_classification': fields.String,
```

Comment:
I don't know why, but these enums worked perfectly fine when using `DAM_MODEL` but exploded the swagger when it was cloned into `DAM_HISTORY_MODEL`. Turned them into `fields.String` on the base model on the off-chance that another enum attribute is added at some point or someone goes to use it as a reference.

Reply:
Interesting.  Should work fine though in this model context.